wagtail/models/__init__.py

```diff
@@ -2832,49 +2794,6 @@ def save(self, user=None, *args, **kwargs):
     def as_object(self):
         return self.content_object.with_content_json(self.content)
 
-    def approve_moderation(self, user=None):
-        warnings.warn(
-            "Revision.approve_moderation() is deprecated and will be removed in a future release.",
-            RemovedInWagtail60Warning,
-            stacklevel=2,
-        )
-        if self.submitted_for_moderation:
-            logger.info(
-                'Page moderation approved: "%s" id=%d revision_id=%d',
-                self.content_object.title,
-                self.content_object.id,
-                self.id,
-            )
-            log(
-                instance=self.as_object(),
-                action="wagtail.moderation.approve",
```

Comment:
I think we also need to remove this (and `wagtail.moderation.reject`) from the list of log actions in `docs/extending/audit_log.md`

Maybe here as well, although we might want to keep them so we can display old logs properly:

https://github.com/wagtail/wagtail/blob/cba75351a8b9512f5fa2b89408879f59f7d064cd/wagtail/wagtail_hooks.py#L130-L131

Reply:
Yep, I think it's better to keep these registered. Have added a comment to mark these as legacy instead.